tox.ini

```diff
@@ -0,0 +1,57 @@
+# tox.ini
+# Runs docker-compose tests. Also creates a development virtualenv.
+# Usage:
+#   tox # runs the test suite
+#   tox -e develop $ create the dev virutalenv
+
+[pypi]
+# Use internal pypi with pre-build wheels for ubuntu
+index = https://registry.affirm-stage.com/artifactory/api/pypi/pypi/simple
+
+[tox]
+envlist = docker-compose, flake8
+minversion = 2.3
+indexserver =
+    default = {[pypi]index}
+# skip setup.py
+skipsdist = True
+
+[testenv]
+basepython = python2.7
+setenv =
+    PIP_INDEX_URL = {[pypi]index}
+    PYTHONIOENCODING = utf-8
+deps =
+  -rtox-requirements.txt
+# for consistency across installations
+sitepackages = False
+whitelist_externals =
+    bin/rm
+    bin/mkdir
+    bin/cp
+    bash
+
+[testenv:develop]
+envdir = .venv
+usedevelop = True
```

Comment:
Does this work with the newer version of tox ? I think they made not having a ```commands``` section illegal. e.g. I had to do this for salt: https://github.com/Affirm/salt/commit/5729e3f25bb73467458cb7679886381b4f5ee2d9